assets/css/oauth-helper-admin.css

```diff
@@ -0,0 +1,54 @@
+.simcal-auth-tabs {
```

Comment:
I know you have just moved file from OAuth helper plugin, but in future try having a SCSS file with a wrapper on top and all the classes and styling inside it, this would help you in making sure the styling is just applied in simple calendar UI only.

Reply:
ok